src/Components/ExternalResult/ResultList.tsx

```diff
@@ -390,36 +388,34 @@ export default function ResultList() {
             </dl>
           </div>
         </div>
-        <div>
-          <div>
-            <div className="text-sm font-semibold mb-2">Search by Name</div>
-            <InputSearchBox
-              search={searchByName}
-              value={qParams.name || ""}
-              placeholder="Search by Patient Name"
-              errors=""
-            />
-          </div>
-          <div>
-            <div className="text-sm font-semibold mt-2">Search by number</div>
-            <InputSearchBox
-              value={qParams.mobile_number || ""}
-              search={searchByPhone}
-              placeholder="Search by Phone Number"
-              errors=""
-            />
+        <div className="w-full">
+          <div className="text-sm font-semibold my-2 lg:my-0 lg:mb-2">
+            Search by Name
           </div>
+          <InputSearchBox
+            search={searchByName}
+            value={qParams.name || ""}
+            placeholder="Search by Patient Name"
+            errors=""
+          />
+          <div className="text-sm font-semibold my-2">Search by number</div>
+          <InputSearchBox
+            value={qParams.mobile_number || ""}
+            search={searchByPhone}
+            placeholder="Search by Phone Number"
+            errors=""
+          />
         </div>
-        <div className="flex flex-col justify-between">
-          <div className="flex">
+        <div className="mt-4 lg:mt-0 ml-auto flex flex-col justify-evenly gap-4">
+          <div className="flex justify-end gap-2">
             <div
-              className="btn mt-8 ml-auto btn-primary"
+              className="btn btn-primary"
               onClick={(_) => navigate("external_results/upload")}
             >
               Upload List
             </div>
             <div
-              className={`btn mt-8 ml-4 gap-2 btn-primary ${
+              className={`btn btn-primary ${
                 downloadLoading ? "pointer-events-none" : ""
               }`}
```

Comment:
```suggestion
              className={clsx("btn btn-primary", downloadLoading && "pointer-events-none")}
```

_can you please use clsx here, it improves the readability and don't forget to import clsx if not already done so_